subgraphs/marketplace/schema.graphql

```diff
@@ -78,3 +88,13 @@ type StakedToken @entity {
   token: Token!
   user: User!
 }
+
+type Stats @entity {
```

Comment:
As discussed in Discord, The Graph doesn't play well with plural entity types. If you try to get all stats:
```
{
  stats {
    id
  }
}
```
You get an error:
> No value provided for required argument: `id`

How about `Data`? `Datas` is kind of strange, but at least it doesn't break 😁 And some of the top subgraphs like Uniswap and Decentraland are using it.